execution_environments/how_to_build_and_test_first_ee.rst

```diff
@@ -0,0 +1,156 @@
+.. _how_to_build_and_test_first_ee:
+
+******************************************************
+How To Build and Test Your First Execution Environment
+******************************************************
+
+* We assume that you installed all :ref:`required packages<how_to_prepare_environment>`.
+* For general information about Execution Environments, see the :ref:`Ansible Execution Environments Overview<ee_overview>` document.
+
+
+.. _build_first_ee:
+
+Building your first EE
+======================
+
+We are going to build an EE containing, in addition to standard packages like ansible-core and Python,
+an Ansible collection (``community.postgresql``) and its dependency (the ``psycopg2`` Python connector).
+
+The collection and the dependency can be replaced with any other package you need.
+
+1. Create and go to a project directory for your new EE:
+
+.. code-block:: bash
+
+  $ mkdir postgresql_ee && cd postgresql_ee
+
+2. Create the ``requirements.yml`` file which will specify collections you want to include in your EE.
```

Comment:
I think we should recommend the single file approach here rather than multiple files.
 e.g. https://github.com/ansible/ansible-builder/blob/devel/demo/execution-environment.yml

Reply:
yep, wasn't sure about best practices. it simplifies the guide, changed, thanks